From: Bernd Zeimetz <bernd@bzed.de>
Date: Sat, 4 Dec 2010 00:05:17 +0100
Subject: [PATCH] Add missing linking for libbluetooth and libusb.

This fixes a FTBFS when gpsd is being built with binutils-gold.

Conflicts:

	Makefile.am
---
 Makefile.am |    5 ++++-
 1 files changed, 4 insertions(+), 1 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index 842082c..ca9dbec 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -120,7 +120,8 @@ lcdgps_LDADD = $(LIBM) libgps.la -lm
 gpsmon_SOURCES = gpsmon.c monitor_nmea.c monitor_sirf.c \
 	monitor_italk.c monitor_ubx.c monitor_superstar2.c \
 	monitor_oncore.c monitor_tnt.c
-gpsmon_LDADD = $(LIBM) $(NCURSES_LIBS) libgpsd.la libgps.la -lm $(LIBPTHREAD)  $(LIBUSB_LIBS)
+gpsmon_LDADD = $(LIBM) $(NCURSES_LIBS) libgpsd.la libgps.la -lm $(LIBPTHREAD)  $(LIBUSB_LIBS) $(BLUEZ_LIBS)
+
 
 #
 # Build gpsdecode
@@ -210,6 +211,8 @@ libgpsd_h_sources = \
 	bits.h \
 	crc24q.h
 
+libgpsd_la_LDFLAGS = $(LIBUSB_LIBS) $(BLUEZ_LIBS)
+
 BUILT_SOURCES = packet_names.h gpsd.h revision.h ais_json.i gps_maskdump.c gpsd_maskdump.c
 
 packet_names.h: packet_states.h
-- 
